mlrun/datastore/targets.py

```diff
@@ -1113,6 +1122,112 @@ def is_single_file(self):
         return True
 
 
+class SnowflakeTarget(BaseStoreTarget):
+    """
+    Snowflake SQL Target
+
+    :param name: Target name
+    :param path:
+    :param attributes: A dictionary of attributes for Snowflake connection; will be overridden by database parameters
+                       if they exist.
+    :param after_step:
+    :param columns:
+    :param partitioned:
+    :param key_bucketing_number:
+    :param partition_cols:
+    :param time_partitioning_granularity:
+    :param max_events:
+    :param flush_after_seconds:
+    :param storage_options:
+    :param schema:
+    :param credentials_prefix:
+    :param url: Snowflake URL, in the format: <account_name>.<region>.snowflakecomputing.com
```

Comment:
This is not a URL though. Either the parameter name is wrong or the example is wrong.

Reply:
Yeah, I think better change this. They shouldn't have called it `sfUrl`, but if you read their docs, they admit that it's not a URL – https://docs.snowflake.com/en/user-guide/spark-connector-use#required-connection-options